frontend/admin-client/src/app/shared/components/pagination/pagination.component.ts

```diff
@@ -7,70 +7,51 @@ import {Component, Input, OnInit, Output, EventEmitter} from '@angular/core';
 })
 export class PaginationComponent implements OnInit {
 
-  @Input() collectionSize: number;
-  @Input() maxSize: number;
+  @Input() currentPage: number;
+  @Input() totalPageCount: number;
+  @Input() visiblePageCount: number;
```

Comment:
Is there a reason you changed the variable names?

Reply:
Yeah, I felt they were confusingly named.  It was especially unintuitive to work with the component without looking at it's innards just because the names didn't really reflect their usage.

Reply:
The visiblePageCount is the only one that could be renamed to be more clear I think